### PR TITLE
Add split strided and translate team example

### PR DESCRIPTION
**Pull request by naveen-rn**
Adding initial example for team split strided and team translate.

### CODE REVIEW

**Comment by naveen-rn** [comment]
Related to https://github.com/gmegan/specification/issues/81

File: example_code/shmem_team_translate.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * shmem_team_split_stride operation
+ */
+
+#include <shmem.h>
+#include <stdio.h>
+
+int main(int argc, char *argv[])
+{
+    int                  rank, npes;
+    int                  t_pe, t_size;
```
Comment (naveen-rn) [inline example_code/shmem_team_translate.c]: Remove unused variables - npes, t_size are unnecessary - add t_global

**Comment by naveen-rn** [comment]
@gmegan This PR is ready to be merged.

**Comment by davidozog** [comment]
Minor suggest on this PR: it seems like `int me = shmem_my_pe()` is used consistently across the specification examples.  `rank` makes one think of MPI :wink:

**Pull request merged**